guide/14-deep-learning/fairness_in_tabulardata_modeling_using_mlmodel.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
```

Comment:
<p>For regression, the <code>mitigation_type</code> can be set to either <code>grid_search</code> or <code>exponentiated_gradient</code> and the <code>mitigation_constraint</code> can be set to <code>ZeroOneLoss</code> or <code>SquareLoss</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2279/discussion/#comment-2461638498'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>